### PR TITLE
Make default output as stdout

### PR DESCRIPTION
**Pull request by ojjiy**
- Set default output as stdout
- some refactorize

### CODE REVIEW

File: combine.py [hunk 1]
```diff
@@ -176,21 +180,31 @@ def main(paths, use_color):
...
 
     total = sum(map(FileInfo, targets))
-    with open('result.txt', 'w') as f:
-        total.save_txt(f, use_color)
+    if output is None:
+        for item in total.to_txt(use_color):
+            print(item)
+    else:
+        with open(output, 'w') as f:
+            f.write('\n'.join(total.to_txt(use_color)))
```
Comment (asi1024) [inline combine.py]: I feel these changes are ugly. How about implementing as follows?
```python
if output_path is None:
    total.save_txt(stdout, use_color)
else:
    with open(output_path, 'w') as f:
        total.save_txt(f, use_color)
```

Comment (asi1024) [reply combine.py]: You can define context manager if you want to unify these `total.save_txt` calls (like as [this site](https://stackoverflow.com/questions/17602878/how-to-handle-both-with-open-and-sys-stdout-nicely)), but I think this solution is too general for this situation.

Comment (ojjiy) [reply combine.py]: Fixed to output only to stdout and rename function

File: readme.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ $ pip install git+https://github.com/ojjiy/line_profiler_extension.git
...
 ```bash
-$ reshape.py [files to be aggregated] [folders which contain files to be aggregated] [--color]
+$ combine.py [files to be aggregated] [folders which contain files to be aggregated] [--color]
 ```
 
 ## Verification
 ```bash
-$ reshape.py ./inputs --colorize
-$ less -R ./result.txt
+$ combine.py ./inputs --colorize
```
Comment (asi1024) [inline readme.md]:
```suggestion
$ combine.py ./inputs --colorize | tee result.out
```

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**